### PR TITLE
Papers/vnet

### PR DESCRIPTION
**Pull request by czotti**
Add the V-Net paper to these categories:
- deep-learning 
- segmentation
- medical

### CODE REVIEW

**Comment by jodoin** [comment]
Please add the full reference to the paper at the beginning (probably : 	F. Milletari, N. Navab, A. Ahmadi 
V-Net: Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation 
International Conference on 3DVision (3DV), Stanford University, California, USA, October 2016)

File: deep-learning/_posts/2017-02-10-vnet.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - http://campar.in.tum.de/pub/milletari2016Vnet/milletari2016Vnet.pdf
+---
+
+<style>
+  main.page-content p {
+    text-align: justify; /* helps the reading flow */
+  }
+</style>
+
+In this paper they address the segmentation of the prostate challenge [PROMISE2012](https://promise12.grand-challenge.org/){:target="_blank"}.
```
Comment (jodoin) [inline deep-learning/_posts/2017-02-10-vnet.md]: Not sure I understand.  The text should be something like

New MRI segmentation method of the prostate based on a 3D CNN similar to the U-Net.  Results reported on the prostate challenge [PROMISE2012](https://promise12.grand-challenge.org

**Comment by jodoin** [comment]
add the CNN tag

File: deep-learning/_posts/2017-02-10-vnet.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+layout: review
+title: V-Net:Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation
+tags: deep-learning CNN segmentation medical
+cite:
+  - V-Net
+  - http://campar.in.tum.de/pub/milletari2016Vnet/milletari2016Vnet.pdf
+---
+
+<style>
+  main.page-content p {
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-02-10-vnet.md]: Please replace `main.page-content` by `div.post-content`. It was a mistake (my mistake) to use `main.page-content` (it adds justification outside the text of the review)

**Comment by jodoin** [comment]
all good

File: deep-learning/_posts/2017-02-10-vnet.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+layout: review
+title: V-Net:Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation
+tags: deep-learning CNN segmentation medical
+cite:
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-02-10-vnet.md]: The cite format has changed. Here is an example of the new format:
```
cite:
    authors: "O. Ronneberger, P. Fischer, T. Brox"
    title:   "U-Net: Convolutional Networks for Biomedical Image Segmentation"
    venue:   "Proceedings of MICCAI 2015, p.234-241"
pdf: "https://arxiv.org/pdf/1505.04597.pdf"
```

**Comment by lemairecarl** [comment]
You can put spaces around the colon in the title, just surround it with quotes:
```
title: "V-Net : Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation"
```

**Pull request merged**